test/TurnUITests.java

```diff
@@ -2,29 +2,99 @@
 import com.TTT.BoardPrinter;
 import com.TTT.Game;
 import com.TTT.TurnUI;
+import org.junit.Before;
 import org.junit.Test;
 
 import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 public class TurnUITests {
+
+    private Board board;
+    private Game game;
+    private BoardPrinter printer;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
I'd suggest avoiding checked exceptions, so having throws Exception should be removed.